docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -71,66 +75,53 @@ Then you will get the binary distribution under **distribution/target**, in whic
 |--------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
-| data\_replication\_factor                  | Data replication factor, no more than DataNode number                                                              |
-| data\_region\_consensus\_protocol\_class   | Consensus protocol of data replicas. Note that RatisConsensus currently does not support multiple data directories |
+| config_node_consensus_protocol_class       | Consensus protocol of ConfigNode                                                                                   |
 | schema\_replication\_factor                | Schema replication factor, no more than DataNode number                                                            |
 | schema\_region\_consensus\_protocol\_class | Consensus protocol of schema replicas                                                                              |
+| data\_replication\_factor                  | Data replication factor, no more than DataNode number                                                              |
+| data\_region\_consensus\_protocol\_class   | Consensus protocol of data replicas. Note that RatisConsensus currently does not support multiple data directories |
 
-Start on Linux:
-```
-# Foreground
-bash ./sbin/start-confignode.sh
-
-# Background
-nohup bash ./sbin/start-confignode.sh >/dev/null 2>&1 &
-```
-
-Start on Windows:
-```
-sbin\start-confignode.bat
-```
-
-More details  [ConfigNode Configurations](https://iotdb.apache.org/UserGuide/Master/Reference/ConfigNode-Config-Manual.html).
+**Notice: The preceding configuration parameters cannot be changed after the cluster is started. Ensure that the common configurations of all Nodes are the same. Otherwise, the Nodes cannot be started.**
 
-### Add ConfigNode (Optional)
+### 5.2.2 ConfigNode configuration
 
-This will add the replication factor of ConfigNode, except for the ports that couldn't conflict with, make sure other configurations are the same with existing ConfigNode in Cluster, and set parameter cn\_target\_config\_nodes\_list as an active ConfigNode in Cluster.
+Open the ConfigNode configuration file ./conf/iotdb-confignode.properties,
+and set the following parameters based on the IP address and available port of the server or VM:
 
-The adding ConfigNode also use the start-confignode.sh/bat.
+| **Configuration**              | **Description**                                                                                                                          |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| cn\_internal\_address          | Internal rpc service address of ConfigNode                                                                                               |
```

Comment:
add a column to descript how to set the value

Reply:
Fixed~